content/docs/quickstart/azure/modify-program.md

```diff
@@ -7,7 +7,9 @@ menu:
     identifier: azure-modify-program
 ---
 
-Let's update our program to do something a little more interesting.
+Now that we have an instance of our Pulumi program deployed, let's enable:
```

Comment:
Add a blank line after this sentence before the beginning of the bullets to make Hugo happy.

Reply:
Yep. Just noticed it too. Will fix it.